xc7/primitives/ff/ff.pb_type.xml

```diff
@@ -17,6 +17,14 @@
   <!-- |FDCE  |      |     |  X  | -->
   <!-- |LDPE  |      |  X  |     | -->
   <!-- |LDCE  |      |  X  |  X  | -->
+  <mode name="NO_FF">
+    <pb_type name="BEL_FF-NO_FF" num_pb="4" blif_model=".subckt NO_FF">
+      <input  name="D" num_pins="1"/>
+    </pb_type>
+    <interconnect>
+      <direct name="D" input="BLK_BB-SLICE_FF.D" output="BEL_FF-NO_FF.D"/>
+    </interconnect>
+  </mode>
```

Comment:
Is it correct not to have any outputs for this `pb_type` (and its `model`)? 
What are the cases in which this `mode` is needed?

Reply:
The first mode in a group is the default mode.  By putting this mode first, it ensure that the CE and SR lines are disconnected, and no FASM features are emitted.  The packer will immediately change modes if needed.

Reply:
Ok, now I get it, thanks